crates/ra_project_model/src/lib.rs

```diff
@@ -183,7 +183,7 @@ impl ProjectWorkspace {
                         if let (Some(&from), Some(&to)) =
                             (sysroot_crates.get(&from), sysroot_crates.get(&to))
                         {
-                            if let Err(_) = crate_graph.add_dep(from, name.into(), to) {
```

Comment:
Let's disable this lint? I find `if let` somewhat more readable, and, with result, `Err(_)` also sends a strong signal that an error is ignored.